content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -312,6 +313,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `CSIDriverRegistry`: Enable all logic related to the CSIDriver API object in csi.storage.k8s.io.
 - `CSIInlineVolume`: Enable CSI Inline volumes support for pods.
 - `CSIMigration`: Enables shims and translation logic to route volume operations from in-tree plugins to corresponding pre-installed CSI plugins
+- `CSIMigrationInTreeOff` : Disables registration of in-tree storage plugins migrated to CSI
```

Comment:
Could you explain a little bit more it's interaction with `CSIMigrationX` flags. (in that it only disables those ones)